tests/integration_test/ota_pal/README.md

```diff
@@ -0,0 +1,19 @@
+#OTA PAL Testing
+
+Please provision your device with the correct certificates in \<root-directory\>/tests/integration_test/ota_pal/test_files before running any OTA PAL tests( these tests are enabled by setting testrunnerFULL_OTA_PAL_ENABLED to 1, see aws_test_runner_config.h ).
```

Comment:
test_files is the name of the directory containing the test files. Did you mean to add <signature-file> to the end of that instead of replacing?

Reply:
I don't see this directory in https://github.com/yanjos-dev/amazon-freertos/tree/yanjos/add_ota_pal_tests/tests/integration_test/ota_pal

Reply:
Thanks for pointing this out. This was previously being ignored by git because of the file path names. I have added .test to the name of the files to clearly point out that they are test files and to avoid them being ignored.

Reply:
The rest of the feedback with putting the tick around files and macros I have added in 2633135.